migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,79 @@
+
+* xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#cluster-health-checklist_premigration-checklists-mtc[Checklist for cluster health]
+* xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#source-cluster-checklist_premigration-checklists-mtc[Checklist for source cluster]
+* xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#target-cluster-checklist_premigration-checklists-mtc[Checklist for target cluster]
+
+[discrete]
+[id='mtc-overview-migrate-mtc-applications']
+== Migrate applications
+xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migrating-applications-with-mtc[Migrate applications]:  You can migrate your applications by using the Migration Toolkit for Containers (MTC) xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migrating-applications-mtc-web-console_migrating-applications-with-mtc[web console] or xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migrating-applications-cli_advanced-migration-options-mtc[the command line]. You can also xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migration-adding-cluster-to-cam_migrating-applications-with-mtc[add the cluster to the MTC web console].
+
+[discrete]
+[id='mtc-overview-advanced-migration-options']
+== Advanced migration options
+xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#advanced-migration-options-mtc[Advanced migration options]: You can automate your migrations and modify the `MigPlan` and `MigrationController` custom resources in order to perform large-scale migrations and to improve performance. You can check the following items:
+
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-creating-registry-route-for-dim_advanced-migration-options-mtc[Create a registry route for direct image migration]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-configuring-proxies_advanced-migration-options-mtc[Configuring proxies]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-migrating-applications-api_advanced-migration-options-mtc[Migrate an application by using the MTC API]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-state-migration-cli_advanced-migration-options-mtc[Use CLI for State migration options]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-hooks_advanced-migration-options-mtc[Check migration hooks options]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-plan-options_advanced-migration-options-mtc[Check migration plan resources]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-controller-options_advanced-migration-options-mtc[Check migration controller options]
```

Comment:
Check migration controller options > Migration controller options for improving performance